app/components/error_summary_component.html.erb

```diff
@@ -0,0 +1,21 @@
+<% # see https://design-system.service.gov.uk/components/error-summary/ %>
+<% content_for :title, flush: true do %>
+  Error: <%= content_for(:title) %>
+<% end %>
+
+<div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
```

Comment:
We shouldn't use `govuk-error-summary` for flash messages, it's meant to be used for forms. Forms already have the error summary via the formbuilder.